frontend/assets/javascripts/src/utils/form/Password.js

```diff
@@ -79,9 +70,9 @@ function ($, bean, component) {
 
         require(['js!zxcvbn'], function() {
 
-            var $passwordStrengthIndicatorTemplate = self.$passwordStrengthIndicatorTemplate();
-
-            $passwordStrengthIndicatorTemplate.insertAfter(self.getElem('PASSWORD_STRENGTH_INPUT'));
+            utilsHelper.toArray(document.querySelectorAll('.' + self.classes.STRENGTH_INDICATOR)).map(function (elem) {
+                elem.classList.toggle('is-hidden');
+            });
```

Comment:
Ah – well, if that's the plan then maybe we should have a card to remove all usages of bonzo/bean etc. and do things natively? Otherwise if we're already loading them IMO we might as well use them – for looping through arrays I think it's nicer to use the syntactic sugar versions...